src/ddscxx/include/org/eclipse/cyclonedds/pub/AnyDataWriterDelegate.hpp

```diff
@@ -105,29 +105,29 @@ class OMG_DDS_API AnyDataWriterDelegate : public org::eclipse::cyclonedds::core:
                           const dds::topic::TopicDescription& td);
 
 
-    inline void setCopyIn(org::eclipse::cyclonedds::topic::copyInFunction copyIn)
+    inline void setCopyIn(org::eclipse::cyclonedds::topic::copyInFunction _copyIn)
     {
-        this->copyIn = copyIn;
+        this->copyIn = _copyIn;
     }
 
     inline org::eclipse::cyclonedds::topic::copyInFunction getCopyIn()
     {
         return this->copyIn;
     }
```

Comment:
These can probably be removed? Don't know if @e-hndrks is removing them in his upcoming work, but these aren't used anymore(?)

Reply:
Removed this function, since it is not used anywhere.